joint_names:
  - prbt_joint_1
  - prbt_joint_2
  - prbt_joint_3
  - prbt_joint_4
  - prbt_joint_5
  - prbt_joint_6

poses:
  # - name: home
  #   positions: [-2.298130859467998e-15, -2.704840289743492e-14, 2.448116797715035e-13, -1.1268166781597117e-13, 2.3466228005892854e-12, 2.0082423675675832e-13]

  - name: pos1
    positions: [0.5225861345857661, -0.1581221816041291, 0.48185462673357565, -0.40451181335024333, 2.5961483155050433, 0.23067926928988272]

  - name: pos3
    positions: [-3.1429135788468e-05, -9.612586560386695e-05, 6.107786855161604e-05, 3.557010920646755e-05, -2.3256274321399995e-06, 6.237134090888349e-06]

  - name: pos4
    positions: [-0.5226569772995896, 0.15809917652817554, -0.3078526495894437, 0.26964039633763376, -2.596147823447346, 1.4022394523193966]

  - name: pos5
    positions: [-1.1463879723479131, 0.5464028538774369, -0.5622269488586272, 0.33722765554028833, -2.292721099938846, 0.3195753985613385]

  - name: pos6
    positions: [1.4160266602806924, -0.5032297899818543, 0.49538826547910547, -2.764840708968444, -1.6352600631453402, 1.1360158628044739]

  - name: pos7
    positions: [1.0114884156628134, -0.5177214810674732, 0.508632829661358, -2.1409383381805642, -1.9554703789081058, 1.4378649617762085]

  # - name: pos8
  #   positions: [6.690054266083566e-05, 8.251356649578077e-05, -5.1792854346176495e-05, -2.041477205886199e-05, -6.185090511969962e-05, 8.31782168492451e-05]
